Minor warnings

GCC 10.1 on Fedora 32
```
In file included from arch/PowerPC/PPCInstPrinter.c:969:
arch/PowerPC/PPCGenAsmWriter.inc: In function "printAliasInstr":
arch/PowerPC/PPCGenAsmWriter.inc:4976: warning: "GETREGCLASS_CONTAIN" redefined
 4976 |   #define GETREGCLASS_CONTAIN(_class, _reg) MCRegisterClass_contains(MCRegisterInfo_getRegClass(MRI, _class), MCOperand_getReg(MCInst_getOperand(MI, _reg)))
      |
arch/PowerPC/PPCInstPrinter.c:715: note: this is the location of the previous definition
  715 | #define GETREGCLASS_CONTAIN(_class, _reg) MCRegisterClass_contains (MCRegisterInfo_getRegClass (MRI, _class), MCOperand_getReg (MCInst_getOperand (MI, _reg)))
      |
arch/X86/X86Module.c: In function "X86_option":
arch/X86/X86Module.c:81:7: warning: multi-line comment [-Wcomment]
   81 | #else // CAPSTONE_DIET \
      |       ^
In file included from tree-sitter/lib/src/././././subtree.h:12,
                 from tree-sitter/lib/src/./././tree_cursor.h:4,
                 from tree-sitter/lib/src/././get_changed_ranges.h:8,
                 from tree-sitter/lib/src/./get_changed_ranges.c:1,
                 from tree-sitter/lib/src/lib.c:8:
tree-sitter/lib/src/./query.c: In function "ts_query__parse_pattern":
tree-sitter/lib/src/./././././array.h:64:42: warning: value computed is not used [-Wunused-value]
   64 | #define array_pop(self) ((self)->contents[--(self)->size])
      |                         ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~
tree-sitter/lib/src/./query.c:755:5: note: in expansion of macro "array_pop"
  755 |     array_pop(&self->steps);
      |     ^~~~~~~~~
class.c: In function "r_bin_java_get_bin_obj_list_thru_obj":
class.c:8143:20: warning: passing argument 2 of "sdb_foreach" from incompatible pointer type [-Wincompatible-pointer-types]
 8143 |  sdb_foreach (sdb, sdb_iterate_build_list, (void *) the_list);
      |                    ^~~~~~~~~~~~~~~~~~~~~~
      |                    |
      |                    int (*)(void *, const char *, const char *)
In file included from ../../libr/include/sdb.h:1,
                 from ../../libr/include/r_list.h:6,
                 from ../../libr/include/r_util.h:10,
                 from class.c:9:
../../libr/include/sdb/sdb.h:133:53: note: expected "SdbForeachCallback" {aka "_Bool (*)(void *, const char *, const char *)"} but argument is of type "int (*)(void *, const char *, const char *)"
  133 | SDB_API bool sdb_foreach(Sdb* s, SdbForeachCallback cb, void *user);
      |                                  ~~~~~~~~~~~~~~~~~~~^~
code.c: In function "r_parse_c_file":
code.c:102:32: warning: passing argument 2 of "sdb_foreach" from incompatible pointer type [-Wincompatible-pointer-types]
  102 |  sdb_foreach (anal->sdb_types, __typeLoad, anal);
      |                                ^~~~~~~~~~
      |                                |
      |                                int (*)(void *, const char *, const char *)
In file included from /home/akochkov/radare/radare2/libr/include/sdb.h:1,
                 from /home/akochkov/radare/radare2/libr/include/r_list.h:6,
                 from /home/akochkov/radare/radare2/libr/include/r_util.h:10,
                 from code.c:3:
/home/akochkov/radare/radare2/libr/include/sdb/sdb.h:133:53: note: expected "SdbForeachCallback" {aka "_Bool (*)(void *, const char *, const char *)"} but argument is of type "int (*)(void *, const char *, const char *)"
  133 | SDB_API bool sdb_foreach(Sdb* s, SdbForeachCallback cb, void *user);
      |                                  ~~~~~~~~~~~~~~~~~~~^~
code.c: In function "r_parse_c_string":
code.c:119:32: warning: passing argument 2 of "sdb_foreach" from incompatible pointer type [-Wincompatible-pointer-types]
  119 |  sdb_foreach (anal->sdb_types, __typeLoad, NULL);
      |                                ^~~~~~~~~~
      |                                |
      |                                int (*)(void *, const char *, const char *)
In file included from /home/akochkov/radare/radare2/libr/include/sdb.h:1,
                 from /home/akochkov/radare/radare2/libr/include/r_list.h:6,
                 from /home/akochkov/radare/radare2/libr/include/r_util.h:10,
                 from code.c:3:
/home/akochkov/radare/radare2/libr/include/sdb/sdb.h:133:53: note: expected "SdbForeachCallback" {aka "_Bool (*)(void *, const char *, const char *)"} but argument is of type "int (*)(void *, const char *, const char *)"
  133 | SDB_API bool sdb_foreach(Sdb* s, SdbForeachCallback cb, void *user);
      |                                  ~~~~~~~~~~~~~~~~~~~^~
p/../arch/8051/8051_disas.c:64:14: warning: "r_8051_disas" defined but not used [-Wunused-function]
   64 | static char *r_8051_disas(ut64 pc, const ut8 *buf, int len, int *olen) {
      |              ^~~~~~~~~~~~
/home/akochkov/radare/radare2/libr/asm/arch/pyc/pyc_dis.c:197:13: warning: "dump_cobj" defined but not used [-Wunused-function]
  197 | static void dump_cobj(pyc_code_object *c) {
      |             ^~~~~~~~~
/home/akochkov/radare/radare2/libr/..//libr/anal/p/anal_arm_gnu.c: In function "op_thumb":
/home/akochkov/radare/radare2/libr/..//libr/anal/p/anal_arm_gnu.c:47:12: warning: overflow in conversion from "long long unsigned int" to "long long int" changes value from "op->val = 18446744073709551615" to "-1" [-Woverflow]
   47 |  op->ptr = op->val = -1;
      |            ^~
/home/akochkov/radare/radare2/libr/..//libr/anal/p/anal_pyc.c: In function "pyc_op":
/home/akochkov/radare/radare2/libr/..//libr/anal/p/anal_pyc.c:64:12: warning: overflow in conversion from "long long unsigned int" to "long long int" changes value from "op->val = 18446744073709551615" to "-1" [-Woverflow]
   64 |  op->ptr = op->val = UT64_MAX;
      |            ^~
/home/akochkov/radare/radare2/libr/..//libr/anal/p/anal_tms320_c55x_plus.c: In function "tms320_c55x_plus_op":
/home/akochkov/radare/radare2/libr/..//libr/anal/p/anal_tms320_c55x_plus.c:28:12: warning: overflow in conversion from "long long unsigned int" to "long long int" changes value from "op->val = 18446744073709551615" to "-1" [-Woverflow]
   28 |  op->ptr = op->val = -1;
      |            ^~
/home/akochkov/radare/radare2/libr/..//libr/anal/p/../..//asm/arch/tms320/c55x_plus/hashvector.c:97:25: warning: "ins_hash" defined but not used [-Wunused-variable]
   97 | static HASHCODE_ENTRY_T ins_hash[] = {
      |                         ^~~~~~~~
class.c: In function "r_anal_class_base_delete_class":
class.c:759:30: warning: passing argument 2 of "r_anal_class_foreach" from incompatible pointer type [-Wincompatible-pointer-types]
  759 |  r_anal_class_foreach (anal, r_anal_class_base_delete_class_cb, &ctx);
      |                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                              |
      |                              int (*)(void *, const char *, const char *)
class.c:134:65: note: expected "SdbForeachCallback" {aka "_Bool (*)(void *, const char *, const char *)"} but argument is of type "int (*)(void *, const char *, const char *)"
  134 | R_API void r_anal_class_foreach(RAnal *anal, SdbForeachCallback cb, void *user) {
      |                                              ~~~~~~~~~~~~~~~~~~~^~
class.c: In function "r_anal_class_base_rename_class":
class.c:784:30: warning: passing argument 2 of "r_anal_class_foreach" from incompatible pointer type [-Wincompatible-pointer-types]
  784 |  r_anal_class_foreach (anal, r_anal_class_base_rename_class_cb, &ctx);
      |                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                              |
      |                              int (*)(void *, const char *, const char *)
class.c:134:65: note: expected "SdbForeachCallback" {aka "_Bool (*)(void *, const char *, const char *)"} but argument is of type "int (*)(void *, const char *, const char *)"
  134 | R_API void r_anal_class_foreach(RAnal *anal, SdbForeachCallback cb, void *user) {
      |                                              ~~~~~~~~~~~~~~~~~~~^~
sign.c: In function "r_sign_find_closest_sig":
sign.c:1181:34: warning: passing argument 2 of "sdb_foreach" from incompatible pointer type [-Wincompatible-pointer-types]
 1181 |  if (!sdb_foreach (a->sdb_zigns, &closest_match_callback, (void *)&data)) {
      |                                  ^~~~~~~~~~~~~~~~~~~~~~~
      |                                  |
      |                                  int (*)(void *, const char *, const char *)
In file included from /home/akochkov/radare/radare2/libr/include/sdb.h:1,
                 from /home/akochkov/radare/radare2/libr/include/r_list.h:6,
                 from /home/akochkov/radare/radare2/libr/include/r_io.h:6,
                 from /home/akochkov/radare/radare2/libr/include/r_anal.h:11,
                 from sign.c:3:
/home/akochkov/radare/radare2/libr/include/sdb/sdb.h:133:53: note: expected "SdbForeachCallback" {aka "_Bool (*)(void *, const char *, const char *)"} but argument is of type "int (*)(void *, const char *, const char *)"
  133 | SDB_API bool sdb_foreach(Sdb* s, SdbForeachCallback cb, void *user);
      |                                  ~~~~~~~~~~~~~~~~~~~^~
pin.c: In function "r_anal_pin_list":
pin.c:87:19: warning: passing argument 2 of "sdb_foreach" from incompatible pointer type [-Wincompatible-pointer-types]
   87 |  sdb_foreach (DB, cb_list, a);
      |                   ^~~~~~~
      |                   |
      |                   int (*)(void *, const char *, const char *)
In file included from /home/akochkov/radare/radare2/libr/include/sdb.h:1,
                 from /home/akochkov/radare/radare2/libr/include/r_list.h:6,
                 from /home/akochkov/radare/radare2/libr/include/r_io.h:6,
                 from /home/akochkov/radare/radare2/libr/include/r_anal.h:11,
                 from pin.c:3:
/home/akochkov/radare/radare2/libr/include/sdb/sdb.h:133:53: note: expected "SdbForeachCallback" {aka "_Bool (*)(void *, const char *, const char *)"} but argument is of type "int (*)(void *, const char *, const char *)"
  133 | SDB_API bool sdb_foreach(Sdb* s, SdbForeachCallback cb, void *user);
      |                                  ~~~~~~~~~~~~~~~~~~~^~
```

--- a/libr/anal/p/anal_arm_gnu.c
+++ b/libr/anal/p/anal_arm_gnu.c
@@ -43,8 +43,6 @@ static int op_thumb(RAnal *anal, RAnalOp *op, ut64 addr, const ut8 *data, int le
 	arm_set_thumb (arminsn, true);
 	arm_set_input_buffer (arminsn, data);
 	arm_set_pc (arminsn, addr);
-	op->jump = op->fail = -1;
-	op->ptr = op->val = -1;
 	op->delay = 0;
 	op->size = arm_disasm_one_insn (arminsn);
 	op->jump = arminsn->jmp;

--- a/libr/anal/p/anal_pyc.c
+++ b/libr/anal/p/anal_pyc.c
@@ -59,9 +59,6 @@ static int pyc_op(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *data, int len, RA
 	ut64 func_base = func->start_offset;
 	ut32 extended_arg = 0, oparg = 0;
 	ut8 op_code = data[0];
-	op->jump = UT64_MAX;
-	op->fail = UT64_MAX;
-	op->ptr = op->val = UT64_MAX;
 	op->addr = addr;
 	op->sign = true;
 	op->type = R_ANAL_OP_TYPE_ILL;

--- a/libr/anal/p/anal_tms320_c55x_plus.c
+++ b/libr/anal/p/anal_tms320_c55x_plus.c
@@ -21,12 +21,8 @@ int tms320_c55x_plus_op(RAnal *anal, RAnalOp *op, ut64 addr, const ut8 *buf, int
 		return 0;
 	}
 
-	memset (op, 0, sizeof (RAnalOp));
-	op->type = R_ANAL_OP_TYPE_UNK;
 	op->addr = addr;
-	op->jump = op->fail = -1;
-	op->ptr = op->val = -1;
-        op->size = ins_len;
+	op->size = ins_len;
 
 	if (ins_len == 1) {
 		if (*ins == 0x20) {

--- a/libr/core/cmd_debug.c
+++ b/libr/core/cmd_debug.c
@@ -1774,7 +1774,6 @@ static int cmd_debug_map(RCore *core, const char *input) {
 					baddr = map->addr;
 
 					if (libname) {
-						char *cmd, *res;
 						const char *file = map->file? map->file: map->name;
 						char *newfile = NULL;
 						if (!r_file_exists (file)) {

